test/utils/navigation.spec.ts

```diff
@@ -196,6 +197,15 @@ describe("NAVIGATION utils", () => {
     expect(navigation).toEqual(`${config.UPDATE_INTERRUPT_CARD_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`);
   });
 
+  test(`NAVIGATION returns ${config.OVERSEAS_ENTITY_QUERY_URL} with 'journey' param set when calling previousPage on ${config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL} object for Remove journey`, () => {
+    const mockRequest = { } as Request;
```

Comment:
Perhaps worth defining this as a `mockRemoveRequest` at top of file as it's also used in the previous test I defined. We can then remove the definition here and in my test method and reduce LOC. It's going to be needed in more test methods going forwards too (I expect).